mimic/tests/test_text_generator.py

```diff
@@ -8,9 +8,9 @@ def setUp(self):
         self.textGenerator = TextGenerator(Model())
     
     def test_load_text_zip(self):
-        pass
-        # self.textGenerator.load_text_zip("zip")
-        # assert something
+        text = self.textGenerator.load_text_zip("../../data/test.zip")
```

Comment:
I don't think there's a test.zip committed

Reply:
I think it's under data/test.zip, at the top of the page. It says binary file not shown, but I think it should still be committed properly. Let me know if it's still not there for you. Thanks.

Reply:
ooh I missed it somehow. all good!